proto/repo.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+import "proto/context.proto";
+
+package repo;
+
+message CreateRepoRequest {
+  // The request context.
+  context.RequestContext request_context = 1;
+
+  // The name of the repo, i.e. "buildbuddy".
+  string name = 2;
+
+  // The organization that the repo should be created under ex. "buildbuddy-io".
+  // If this is empty, the repo will be created under the authenticated Github
+  // User's account.
+  string organization = 3;
+
+  // A short description of the repo, ex. "BuildBuddy is an open source Bazel
+  // build event viewer, result store, remote caching, and remote build
+  // execution platform."
+  string description = 4;
+
+  // An existing git repository to use as a template for the contents of the new
+  // repo. ex.
+  // "https://github.com/buildbuddy-io/go-typescript-starter-template.git"
```

Comment:
nit: maybe say what we do with the template / how it's used?

Reply:
Done